README.md

```diff
@@ -100,6 +100,7 @@ _Note: Links are provided further down to create an app for your logged in user
 
 * Actions (read)
 * Administration (read / write)
+* Checks (read) (if you are going to use [Faster Autoscaling with GitHub Webhook](#faster-autoscaling-with-github-webhook))
```

Comment:
Oh, really? I was in an impression that we are only relying on webhook payload of check run events so there's no need to call any GitHub API to access checks, hence no need to add the permission.

Reply:
> we are only relying on webhook payload

aren't we?

Reply:
What do you mean? AFAIK, here's the only place in where we rely on check runs. You'll see there's no API calls related to check runs?

https://github.com/actions-runner-controller/actions-runner-controller/blob/f858e2e43273a861b06b546c6311fe68fb6e1ce5/controllers/horizontal_runner_autoscaler_webhook_on_check_run.go#L9

Reply:
It's impossible to subscribe to "Check run" events at all if you don't grant read access to Checks

Reply:
@Tarasovych Oh, that's good to know! Thanks for your clarification ☺️